src/main/java/uk/gov/hmcts/ccd/SecurityConfiguration.java

```diff
@@ -1,63 +1,102 @@
 package uk.gov.hmcts.ccd;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.security.authentication.AuthenticationManager;
-import org.springframework.security.authentication.ProviderManager;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.builders.WebSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
-import uk.gov.hmcts.reform.auth.checker.core.RequestAuthorizer;
-import uk.gov.hmcts.reform.auth.checker.core.service.Service;
-import uk.gov.hmcts.reform.auth.checker.core.user.User;
-import uk.gov.hmcts.reform.auth.checker.spring.serviceanduser.AuthCheckerServiceAndUserFilter;
+import org.springframework.security.oauth2.core.DelegatingOAuth2TokenValidator;
+import org.springframework.security.oauth2.core.OAuth2TokenValidator;
+import org.springframework.security.oauth2.jwt.*;
+import org.springframework.security.oauth2.server.resource.web.BearerTokenAuthenticationFilter;
+import uk.gov.hmcts.ccd.security.JwtAuthorityExtractor;
+import uk.gov.hmcts.ccd.security.filters.PathParamSecurityFilter;
+import uk.gov.hmcts.ccd.security.filters.ServiceAuthFilter;
+import uk.gov.hmcts.reform.authorisation.validators.AuthTokenValidator;
 
 import javax.inject.Inject;
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
 
 import static org.springframework.security.config.http.SessionCreationPolicy.STATELESS;
 
 @Configuration
 @EnableWebSecurity
 public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
 
-    private final AuthCheckerServiceAndUserFilter authCheckerFilter;
+    @Value("${spring.security.oauth2.client.provider.oidc.issuer-uri}")
+    private String issuerUri;
+
+    @Value("${oidc.issuer}")
+    private String issuerOverride;
+
+    private final ServiceAuthFilter serviceAuthFilter;
+    private final PathParamSecurityFilter pathParamSecurityFilter;
```

Comment:
this filter is only used for v1 endpoints correct? if yes I'd suggest we make that more explicit in the name, like
`V1EndpointsSecurityFilter` or `V1EndpointsPathParamSecurityFilter`